src/lib/isTaxID.js

```diff
@@ -1211,30 +1215,62 @@ const sanitizeRegexes = {
 // sanitizeRegexes locale aliases
 sanitizeRegexes['nl-BE'] = sanitizeRegexes['fr-BE'];
 
+const multipleTinLocale = {
+  'en-IN': {
+    GSTIN: true,
+    PAN: true,
+  },
+};
+
+const availableOptions = {
+  localeOption: true,
+};
+
+// Validates args of isTaxID function
+function validateArgs(str, locale, options) {
+  const localeOption = options?.localeOption;
+
+  try { assertString(str); } catch (err) { throw new Error(`${err.message} for str`); }
+  try { assertString(locale); } catch (err) { throw new Error(`${err.message} for locale`); }
+
+  for (const option in options) {
+    if (!(option in availableOptions)) throw new Error(`'${option}' is not available`);
+  }
+
+  if (!(locale in taxIdFormat)) throw new Error(`Invalid locale '${locale}'`);
+
+
+  if (locale in multipleTinLocale) {
+    try { assertString(localeOption); } catch (err) { throw new Error(`${err.message} for localeOption`); }
+    if (!(localeOption in multipleTinLocale[locale])) throw new Error(`Invalid localeOption '${localeOption}'`);
+  } else if (localeOption || localeOption === '') {
+    throw new Error(`Invalid localeOption for locale '${locale}'`);
+  }
+}
+
```

Comment:
Did you get inspiration for this from some other validator using an `options` object or did you come up with this yourself? To me this seems a bit heavy handed. Are we generally interested in throwing errors if the user doesn't provide valid options? I haven't seen this in other validators. Does the multiple locale possibility in `en-IN` warrant this much code here?

#2086 is somewhat similar to this PR in that it uses a local option in the options object. Maybe cut things down a bit, taking inspiration from there?

Reply:
Hey @braaar  thanks for your feedback,
I came up with this approach, current function throws error if user specifies a wrong `locale` or `str`, so I thought it might be useful if we throw error for wrong `options` too. And yeah since `localeOption` is dependent on `locale` it seems a bit heavy handed,even I had the same thought too. 
But wouldn't it be misleading if the validation returns `false` when incorrect localeOption is passed ?

For now, `en-IN` is the only locale which has multiple TIN, but when I researched through the internet there are several other countries who have multiple TIN.

Example : [US TIN](https://www.irs.gov/individuals/international-taxpayers/taxpayer-identification-numbers-tin)
US has 5 different Identification numbers.

Reply:
This complexity is starting to make me feel like maintaining a validator for multiple tax ids for multiple countries is an enourmous undertaking. I suppose it's the only natural way forward, unless we decide to scrap the validator altogether, of course. 

How many app developers are making international systems that care about tax ids and are using open source validation code? In my eyes the use case for such an all-encompassing validator is quite rare. When I work with things like social security numbers and business IDs I use a [country-specific package](https://www.npmjs.com/package/norwegian-numbers) that contains what I need and is much easier to maintain on its own, should I spot an error or want something added. 

What are your thoughts on this aspect, @WikiRik? I suppose this enters into a territory where the maintainers should chime in. They are in control of the main direction of the project and also feel the burden of maintaining code the most.

Reply:
Yes, input from the maintainers is indeed something we'll need. But the fact is that we already have `isTaxID` with multiple countries so this is a logical addition in my view.

Reply:
@profnandaa can you please help us here.

Reply:
Perhaps I should open a separate issue about scrapping this validator.

Reply:
yeah and can we just add PAN regex alone for en-IN locale ,for now ?